### ISSUE TITLE
Login issue

### ISSUE THREAD

**Issue by silentvick** [paimpozhil/DockerX2go#4]
Hi,

I've been playing around with your `xubuntu` image recently. And I managed to fix the issue about need to login as `root` first. I found that the problem is that `/tmp/.X11-unix` doesn't exist. I guess that's why X server can't start. When you're trying to start session as unprivileged user it fails, because you don't have permission to create that directory. When you login as root the `.X11-unix` gets created, and then you can start session as usual.

Also I found that it is somehow related to upgrading in your build. Drop the `apt-get upgrade` line and all will work just as expected.

So, I see two options here:
1. Stop doing `upgrade` in your build. Because, do you realy need it? Maybe it's better to leave it on maintainers of base images (e.g. `ubuntu` that you based on)?
2. Create the `/tmp/.X11-unix` directory manually. For example, add this in your Dockerfile:
   
   ```
   RUN mkdir -p /tmp/.X11-unix && chmod 1777 /tmp/.X11-unix
   ```

_Also, you have to add `dockerx` to `x2gouser` group. You have that for `lxde` but not for `xubuntu`._

**Comment by paimpozhil** [comment]
@silentvick Thanks for the root-login-first issue fix :+1: 

Actually I found that Lxde / Centos images were more light weight than the Xubuntu so I didnt update/use this often.

This image is one of my very first docker images that I created so obviously have some brain damages. Apt-get upgrade is one of them..

I will update all the suggested changes.. 

PR would have been better however these are tiny So I will incorporate them now .

Thanks for working/ helping with this.
